docs/docs/advanced/air-gap.md

```diff
@@ -9,7 +9,11 @@ At first, you need to download the vulnerability database for use in air-gapped
 Please follow [oras installation instruction][oras].
 
 Download `db.tar.gz`:
+```
+$ oras pull ghcr.io/aquasecurity/trivy-db:2
+```
 
+for `oras` v0.12.0 or earlier, `-a` flag must be used:
```

Comment:
Can we use "grouping code blocks"?
https://squidfunk.github.io/mkdocs-material/reference/content-tabs/#linked-content-tabs

```suggestion
=== oras >= v0.13.0
     ```
     $ oras pull ghcr.io/aquasecurity/trivy-db:2
     ```

=== oras < v0.13.0
     ```
     $ oras pull -a ghcr.io/aquasecurity/trivy-db:2
     ```
```

Reply:
Done.